azkaban-web-server/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -671,6 +676,11 @@ private void ajaxScheduleFlow(HttpServletRequest req,
     }
 
     List<SlaOption> slaOptions = null;
+    try {
+      slaOptions = getSlaOptionList(req, flowName);
+    } catch (Exception e) {
```

Comment:
The getSlaOptionList throws a serveletException already. 
Could you explain why you would like to use a different error handling strategy?